tests/python/cli/example_function.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023 CVAT.ai Corporation
```

Comment:
I suppose this file can be used as an example by many people, so I suggest improving the style and naming to spread good behaviors. I'd suggest to add type annotations to the function. I'm not sure we should really use the same low level API models, since we don't really request anything here, and changes there may easily break user code for no reason. Maybe we can just define no-op inherited classes just to expose them in this API as the first step to decoupling.

Reply:
> I suppose this file can be used as an example by many people, so I suggest improving the style and naming to spread good behaviors. I'd suggest to add type annotations to the function.

I can add types, sure. I'm not sure if this is a good example to follow, though... I think a YOLOv8-based function that @nmanovic suggested in another thread would be a better example.

>  I'm not sure we should really use the same low level API models, since we don't really request anything here, and changes there may easily break user code for no reason.

I mean... that's why we shouldn't make such changes. Naturally, breaking changes cause breakage.

Reply:
Type annotations added.